src/icons/Checkmark.tsx

```diff
@@ -22,12 +22,12 @@ export default class Checkmark extends React.PureComponent<Props> {
       <Svg
         width={this.props.width}
         height={this.props.height}
-        viewBox="0 0 32 32"
+        viewBox="0 0 16 16"
         fill="none"
         testID={this.props.testID}
       >
         <Path
-          d="M8.414 15.086a2 2 0 10-2.828 2.828l2.828-2.828zM12.5 22l-1.414 1.414a2 2 0 002.8.029L12.5 22zm13.885-10.557a2 2 0 00-2.77-2.886l2.77 2.886zm-20.8 6.471l5.5 5.5 2.83-2.828-5.5-5.5-2.83 2.828zm8.3 5.529l12.5-12-2.77-2.886-12.5 12 2.77 2.886z"
+          d="M13.667 4.886 6.124 12.43 2.667 8.972l.886-.886 2.57 2.564L12.78 4l.887.886Z"
```

Comment:
I intentionally updated current shared icons/components according to new designs: if we got a new look for `Checkmark`, let's consider that look should be applied everywhere.

Otherwise, we shouldn't use shared elements and implement everything for each screen separately.

Or pollute shared folders with Checkmark2, Checkmark3, CheckmarkWithBlowsAndWhistles etc.

Let me check with Nitya to see if she is okay with my proposed approach. If not, I will revert those in favor of dedicated icons for those new designs.

Reply:
I've checked with @nityarao yesterday and she approved that we can update icons globally. 

Specifically for the `Checkmark`, I have updated it as per designs: https://www.figma.com/file/erFfzHvSTm5g1sjK6jWyEH/Working-Design-System-Components?type=design&node-id=585-17&mode=design&t=fihuWlOHIEwfJsx1-0

Note that sizes remain intact.